package/src/components/AutoCompleteInput/AutoCompleteSuggestionHeader.tsx

```diff
@@ -41,60 +44,40 @@ const AutoCompleteSuggestionHeaderWithContext = ({
     },
   } = useTheme();
 
+  return (
+    <View style={[styles.container, container]}>
+      <Smile pathFill={accent_blue} />
+      <Text style={[styles.title, { color: grey }, title]} testID='emojis-header-title'>
+        {`Emoji matching "${queryText}"`}
+      </Text>
+    </View>
+  );
+};
+
+export const AutoCompleteSuggestionHeader = ({
+  queryText,
+  triggerType,
+}: AutoCompleteSuggestionHeaderProps) => {
   if (triggerType === 'command') {
-    return (
-      <View style={[styles.container, container]}>
-        <Lightning fill={accent_blue} size={32} />
-        <Text style={[styles.title, { color: grey }, title]} testID='commands-header-title'>
-          {t<string>('Instant Commands')}
-        </Text>
-      </View>
-    );
+    return <CommandsHeader />;
   } else if (triggerType === 'emoji') {
-    return (
-      <View style={[styles.container, container]}>
-        <Smile pathFill={accent_blue} />
-        <Text style={[styles.title, { color: grey }, title]} testID='emojis-header-title'>
-          {t<string>('Emoji matching') + ' "' + queryText + '"'}
-        </Text>
-      </View>
-    );
-  } else if (triggerType === 'mention') {
-    return null;
+    return <EmojiHeader queryText={queryText} />;
   } else {
     return null;
   }
 };
 
-const areEqual = (
```

Comment:
same here

Reply:
Pretty much the same reason as above. I don't see any benefit/advantage of the change, but it just makes more sense to me. Happy to change it if you think memo can be advantageous here.

Reply:
I have added it back. Will think about these improvements later so that I don't mess things up here.